templates/page.wrr.json

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
Auch bei WRR, wusste ich nicht was das ist. Vielleicht "right-to-return" oder "right-on-refund"?